src/frontend/overlay/src/components/atoms/ContestLabels.jsx

```diff
@@ -115,27 +117,52 @@ const TaskResultLabelWrapper2 = styled.div`
   color: #fff;
 `;
 
-// TODO: fts start
-const ICPCTaskResultLabel2 = ({ problemResult: r, ...props }) => {
+
+const StarIconWrap = styled.div`
+    position: absolute;
+    width: 33px;
+    height: 33px;
+    mask: url(${star});
+    background-position: center;
+    background-color: ${({ color }) => color};
+`;
+
+const StarIcon = ({ color }) => {
```

Comment:
There is no reason for StarIcon component. Just rename StarIconWrap to StarIcon

Reply:
fixed